portfolio/src/main/webapp/js/comments.js

```diff
@@ -14,7 +33,9 @@ function loadComments() {
     })
     .catch(err => {
       console.error(err);
-      alert("Failed to populate comments.");
+
+      addNotification("Failed to populate the comments section!", 
+        "alert-danger");
```

Comment:
if we want to wrap, indentation should be 4 spaces https://google.github.io/styleguide/jsguide.html#formatting-function-arguments

Also, in the 3 examples in this section, I prefer the first or the third. As it states, the second, which is similar to what's here, is discouraged.

Reply:
Oh wow. You caught a huge oversight on my part here, thanks. Will update.

Reply:
@billyeh How do things look now?